.github/actions/5a-linux-arm64/action.yml

```diff
@@ -0,0 +1,53 @@
+name: "Linux: Cross-compile ARM64 libraries & copy to install dir"
+runs:
+  using: composite
+  steps:
+
+    - name: Install Cross-Compile Support (ARM64)
+      uses: cyberjunk/gha-ubuntu-cross@v5
+      with:
+        arch: arm64
+
+          #    - name: Install libx11-dev for ARM64
+          #run: sudo apt-get install libx11-dev:arm64
+
+    - shell: bash
+      run: |
+        set -eux
+        cd ..
+
+        # keep using our own lld as a linker
+        sudo rm /usr/bin/aarch64-linux-gnu-ld
+        sudo ln -sf /usr/bin/ld /usr
+
+        arch='aarch64'
+        triple="$arch-linux-gnu"
+
+        export CC=clang
+        ./installed/bin/ldc-build-runtime --ninja \
+          --buildDir="build-libs-$arch" \
+          --dFlags="-mtriple=$triple -gcc=clang" \
+          --cFlags="--target=$triple" \
+          --linkerFlags="--target=$triple" \
+          --ldcSrcDir="$PWD/ldc"
+
+        mkdir "installed/lib-$arch"
+        cp -rv "build-libs-$arch"/lib/* "installed/lib-$arch/"
+
+        section="
+        \"($arch|arm64).*-linux-gnu\":
+        {
+            switches = [
+                \"-defaultlib=phobos2-ldc,druntime-ldc\",
+                \"--mcpu=neoverse-n1\",
```

Comment:
is setting the CPU really needed?  (seems better to keep the runtime build more general as it probably won't really matter whether it uses newer CPU instructions or not)

Reply:
But that's not for the runtime build, that's the default flags to use then building for ARM64. Since Weka requires these settings and this is a Weka-specific compiler package, I thought it makes sense to add them to the config.

But of course one could argue that we should keep these settings in the Wekapp repo internally.

I'm fine both ways. I think it only matter if we want to try merging these changes upstream. If it's going to stay Weka-specific, seems to better keep the required flags in the config.